agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -910,16 +910,16 @@ func TestTwoTasksSharedLocalVolume(t *testing.T) {
 	wTask, err := agent.StartTask(t, "task-shared-vol-write")
 	require.NoError(t, err, "Register task definition failed")
 
-	// then reader task
-	rTask, err := agent.StartTask(t, "task-shared-vol-read")
-	require.NoError(t, err, "Register task definition failed")
-
-	// clean up
+	// Wait for the first task to create the volume
 	wErr := wTask.WaitStopped(waitTaskStateChangeDuration)
```

Comment:
Should we just wait until the first task transitions to `RUNNING` instead of `STOPPED`?

Reply:
Yes, this just makes the change easier. Probably doing this in another PR.